npm-shrinkwrap.json

```diff
@@ -1821,7 +1821,7 @@
         "babel-code-frame": {
           "version": "6.22.0",
           "from": "babel-code-frame@>=6.22.0 <7.0.0",
-          "resolved": "https://registry.npmjs.org/babel-code-frame/-/babel-code-frame-6.22.0.tgz"
+          "resolved": "http://registry.npmjs.org/babel-code-frame/-/babel-code-frame-6.22.0.tgz"
```

Comment:
Beats me why this is happening. But this was my third and most successful attempt at updating the shrinkwrap file.